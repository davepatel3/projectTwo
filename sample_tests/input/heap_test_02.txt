7049
create_file_root
4
O ycRtE6B
create_file_root
125
Yi0-w
get_available_size
create_file
39
FR1hk5_I8pNoIns4w
0
exist
3
create_directory
FEUcwelUp0i-5kW
0
get_handle
/FEUcwelUp0i-5kW
print_layout
get_absolute_path
2
create_directory
XY2cftnVZ8i4CUCWOT
4
create_directory
NC8AtyPN
4
print_layout
exist
5
print_layout
create_directory
mNy3ZdcCMxp
5
get_largest_file_handle
get_absolute_path
2
get_available_size
create_file
8
OKrB8IWg3peND4E-KJps
4
create_file
201
SgvI91vIn
4
create_file
37
H1 ZaH_
6
create_file
10
qyCs5j 
7
create_directory
KrVcFRlL_ASFL
5
create_file
14
uzIOY7X xnSqX
12
exist
11
create_file
11
x22qVMWBvNYcHF
7
get_name
7
create_file
6
DMMkR38JHz9r
12
create_file
25
HGMxmgpDEk5kuK
7
create_directory
v0 nDfqV0z9y
7
get_available_size
create_file
34
JWu34ktvRE
6
print_layout
create_directory_root
-At4o
create_directory
wCTAnDVOfM3v
4
get_available_size
get_largest_file_handle
get_absolute_path
19
create_directory
LUFvyHAEdZw2RVv
5
print_layout
create_directory
9gWzvIjm
12
create_file
198
hAhye-21sTybw
12
create_directory
OkkCCClxg
5
get_largest_file_handle
get_handle
/FEUcwelUp0i-5kW/OKrB8IWg3peND4E-KJps
get_largest_file_handle
get_absolute_path
17
get_handle
/FEUcwelUp0i-5kW/XY2cftnVZ8i4CUCWOT/LUFvyHAEdZw2RVv
get_largest_file_handle
get_largest_file_handle
get_file_size_path
/FEUcwelUp0i-5kW/XY2cftnVZ8i4CUCWOT/KrVcFRlL_ASFL/hAhye-21sTybw
create_directory
1cT17e3thA u
4
create_file
104
j_93WK
22
print_layout
get_name
16
get_largest_file_handle
get_available_size
create_directory
G1Tp
7
create_file_root
29
vyXqjk-9v62b7Ll8sb
get_available_size
create_file
128
mzxgp-eYVkRBuPyd
4
create_directory
uct U0gQoMdMO
4
get_available_size
get_available_size
quit
