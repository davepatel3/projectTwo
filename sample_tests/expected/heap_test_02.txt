create_file_root
1
create_file_root
2
get_available_size
6920
create_file
3
exist
1
create_directory
4
get_handle
4
print_layout
[F]O ycRtE6B (size = 4)
[F]Yi0-w (size = 125)
[F]FR1hk5_I8pNoIns4w (size = 39)
[D]FEUcwelUp0i-5kW

get_absolute_path
/Yi0-w
create_directory
5
create_directory
6
print_layout
[F]O ycRtE6B (size = 4)
[F]Yi0-w (size = 125)
[F]FR1hk5_I8pNoIns4w (size = 39)
[D]FEUcwelUp0i-5kW
	[D]XY2cftnVZ8i4CUCWOT
	[D]NC8AtyPN

exist
1
print_layout
[F]O ycRtE6B (size = 4)
[F]Yi0-w (size = 125)
[F]FR1hk5_I8pNoIns4w (size = 39)
[D]FEUcwelUp0i-5kW
	[D]XY2cftnVZ8i4CUCWOT
	[D]NC8AtyPN

create_directory
7
get_largest_file_handle
2
get_absolute_path
/Yi0-w
get_available_size
6881
create_file
8
create_file
9
create_file
10
create_file
11
create_directory
12
create_file
13
exist
1
create_file
14
get_name
mNy3ZdcCMxp
create_file
15
create_file
16
create_directory
17
get_available_size
6569
create_file
18
print_layout
[F]O ycRtE6B (size = 4)
[F]Yi0-w (size = 125)
[F]FR1hk5_I8pNoIns4w (size = 39)
[D]FEUcwelUp0i-5kW
	[D]XY2cftnVZ8i4CUCWOT
		[D]mNy3ZdcCMxp
			[F]qyCs5j  (size = 10)
			[F]x22qVMWBvNYcHF (size = 11)
			[F]HGMxmgpDEk5kuK (size = 25)
			[D]v0 nDfqV0z9y
		[D]KrVcFRlL_ASFL
			[F]uzIOY7X xnSqX (size = 14)
			[F]DMMkR38JHz9r (size = 6)
	[D]NC8AtyPN
		[F]H1 ZaH_ (size = 37)
		[F]JWu34ktvRE (size = 34)
	[F]OKrB8IWg3peND4E-KJps (size = 8)
	[F]SgvI91vIn (size = 201)

create_directory_root
19
create_directory
20
get_available_size
6535
get_largest_file_handle
9
get_absolute_path
/-At4o
create_directory
21
print_layout
[F]O ycRtE6B (size = 4)
[F]Yi0-w (size = 125)
[F]FR1hk5_I8pNoIns4w (size = 39)
[D]FEUcwelUp0i-5kW
	[D]XY2cftnVZ8i4CUCWOT
		[D]mNy3ZdcCMxp
			[F]qyCs5j  (size = 10)
			[F]x22qVMWBvNYcHF (size = 11)
			[F]HGMxmgpDEk5kuK (size = 25)
			[D]v0 nDfqV0z9y
		[D]KrVcFRlL_ASFL
			[F]uzIOY7X xnSqX (size = 14)
			[F]DMMkR38JHz9r (size = 6)
		[D]LUFvyHAEdZw2RVv
	[D]NC8AtyPN
		[F]H1 ZaH_ (size = 37)
		[F]JWu34ktvRE (size = 34)
	[F]OKrB8IWg3peND4E-KJps (size = 8)
	[F]SgvI91vIn (size = 201)
	[D]wCTAnDVOfM3v
[D]-At4o

create_directory
22
create_file
23
create_directory
24
get_largest_file_handle
9
get_handle
8
get_largest_file_handle
9
get_absolute_path
/FEUcwelUp0i-5kW/XY2cftnVZ8i4CUCWOT/mNy3ZdcCMxp/v0 nDfqV0z9y
get_handle
21
get_largest_file_handle
9
get_largest_file_handle
9
get_file_size_path
198
create_directory
25
create_file
26
print_layout
[F]O ycRtE6B (size = 4)
[F]Yi0-w (size = 125)
[F]FR1hk5_I8pNoIns4w (size = 39)
[D]FEUcwelUp0i-5kW
	[D]XY2cftnVZ8i4CUCWOT
		[D]mNy3ZdcCMxp
			[F]qyCs5j  (size = 10)
			[F]x22qVMWBvNYcHF (size = 11)
			[F]HGMxmgpDEk5kuK (size = 25)
			[D]v0 nDfqV0z9y
		[D]KrVcFRlL_ASFL
			[F]uzIOY7X xnSqX (size = 14)
			[F]DMMkR38JHz9r (size = 6)
			[D]9gWzvIjm
				[F]j_93WK (size = 104)
			[F]hAhye-21sTybw (size = 198)
		[D]LUFvyHAEdZw2RVv
		[D]OkkCCClxg
	[D]NC8AtyPN
		[F]H1 ZaH_ (size = 37)
		[F]JWu34ktvRE (size = 34)
	[F]OKrB8IWg3peND4E-KJps (size = 8)
	[F]SgvI91vIn (size = 201)
	[D]wCTAnDVOfM3v
	[D]1cT17e3thA u
[D]-At4o

get_name
HGMxmgpDEk5kuK
get_largest_file_handle
9
get_available_size
6233
create_directory
27
create_file_root
28
get_available_size
6204
create_file
29
create_directory
30
get_available_size
6076
get_available_size
6076
quit
